site/src/pages/WorkspacesPage/WorkspacesPageView.tsx

```diff
@@ -96,15 +97,19 @@ export const WorkspacesPageView: FC<
           presetFilters={presetFilters}
         />
       </Stack>
-
       <WorkspacesTable
-        isLoading={isLoading}
-        workspaceRefs={workspaceRefs}
-        filter={filter}
-        isNonInitialPage={isNonInitialPage}
+        workspaces={workspaces}
+        isUsingFilter={filter !== workspaceFilterQuery.me}
+        onUpdateWorkspace={onUpdateWorkspace}
       />
-
-      <PaginationWidget numRecords={count} paginationRef={paginationRef} />
+      {count && (
```

Comment:
I think we might need `Boolean(count)` or a ternary or embed the zero logic into the component itself or something like that because I think when this is zero it displays a zero on the page.

Reply:
That is true, I've been seeing this happening some times.